pkg/plugin/linuxutil/netstat_stats_linux.go

```diff
@@ -22,6 +22,21 @@ const (
 	pathNetSnmp    = "/proc/net/snmp"
 )
 
+var (
+	nodeIP     = os.Getenv("NODE_IP")
+	ignoreList = []string{
+		"0.0.0.0",
+		"127.0.0.",
```

Comment:
Missing 0 at the end.

Reply:
using hasprefix so that it matches localhost addresses in the 127.0.0.0/8 range